hyrax/app/assets/stylesheets/theme.scss

```diff
@@ -12,6 +12,9 @@
   font-style: normal;
 }
 
+//------------ Variables ---------------
+$footer-height: 200px;
```

Comment:
$footer-height defined here is not being used and the rendered footer height is still 118px, from the upstream stylesheets. This is probably what's happening:

1. _variables.scss sets $footer-height 
2. _footer.scss sets body margin-bottom and .site-footer height
3. theme.scss sets $footer-height
4. The new value never gets used

Reply:
If we look at Rails.application.config.assets.paths, the assets in this application comes first, so the above comment is likely wrong and does not describe what's happening.

Reply:
So to perform the same arithmetic operations as the upstream's _footer.scss, bootstrap must be loaded in this file.